### ISSUE TITLE
SECKSD-22577 `akamai_networklist_activations` does not support terraform import

### ISSUE THREAD

**Issue by gsilvapt** [akamai/terraform-provider-akamai#469]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
```shell
Terraform v1.5.6
on darwin_arm64
+ provider registry.terraform.io/akamai/akamai v5.2.0
```

### Affected Resource(s)
- akamai_networklist_activations

### Terraform Configuration Files
Not relevant to the issue as I'm following the guide [here](https://techdocs.akamai.com/terraform/docs/set-up-network-lists#import-a-network-list) to import a list into the state.

### Debug Output

After running `terraform import "akamai_networklist_activations.activation[\"List Name\"] 0000000":
```
akamai_networklist_activations.activation["List Name"]: Importing from ID "0000000"...
╷
│ Error: resource akamai_networklist_activations doesn't support import
```

### Panic Output
N/A

### Expected Behavior

When importing existing NetworkLists from the account into Terraform, just as with `akamai_networklist_network_list`, it is desired to be able to import the corresponding activations in the Terraform state. As things are, we are able to import the lists into the state with `terraform import akamai_networklist_network_list[key] [uniqueid]` but the plan will always create the corresponding activations. This can lead to undesired activations really, and the activation should only be triggered when there is a change in the list.

### Actual Behavior

To import existing `NetworkLists` into Terraform, we have to recreate the activations in the state with `terraform apply`, as it 's not possible to leverage `terraform import` to get them into the sate, meaning we will be making uncontrolled API calls under the hood which can cause undesired changes in the platform. 

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Re-create an existing list with Terraform code. I used a custom script to fetch data from the API but choose an isolated case. Copy the values into the resource block.
2. Import the list to your state with `terraform import akamai_networklist_network_list <uniqueid>`, where `uniqueid` is obtained in the API.
3. Notice your `terraform plan` will still suggest to create the activation.
4. If you get the last activation data from the API, you cannot import it into state, and the plan will suggest to create it.
5. If you run `terraform apply`, there could be calls changing your list state underneath, leading to undesired changes.

### Important Factoids
N/A

### References
N/A

**Comment by JoshWeepie** [comment]
Yes, this should work exactly the same as property activations with the ability to import

**Comment by JoshWeepie** [comment]
I spent time testing this today, this is ridiculous. Essentially it's impossible to import existing network lists, because activating a new version of a list without making any changes to the list will also fail. 

So there is no way to import existing network lists and their corresponding staging/prod activations. Basically the akamai_networklist_network_list resource can only be used for new lists and if you are a legacy customer you are left out to dry and forced to upgrade to client lists in order to just import IP lists.
 
The only way it seems to work around this is to add a fake IP to your list, activate it in staging/prod, and then remove the IP and activate yet another version of the network list.
